### PR TITLE
Added brazilian portuguese translation

### PR DESCRIPTION
**Pull request by rutantan**

### CODE REVIEW

File: pegasus_pt-br.ts [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/SettingsMain.qml" line="106"/>
+        <source>General</source>
+        <translation>Geral</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/SettingsMain.qml" line="115"/>
+        <source>Language</source>
+        <translation type="unfinished"></translation>
```
Comment (mmatyas) [inline pegasus_pt-br.ts]: The translation seems to be missing here.

File: translations.qrc.in [hunk 2]
```diff
@@ -4,6 +4,7 @@
         <file>pegasus_es.qm</file>
         <file>pegasus_fr.qm</file>
         <file>pegasus_hu.qm</file>
+		<file>pegasus_pt-br.qm</file>
```
Comment (mmatyas) [inline translations.qrc.in]: Use spaces here for the indentation.

File: pegasus_pt-br.ts [hunk 3]
```diff
@@ -0,0 +1,464 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mmatyas) [inline pegasus_pt-br.ts]: The file should be called `pegasus_pt-BR.ts` (ie. capital `BR`).

**Review by mmatyas — CHANGES_REQUESTED** [review]
Thank you! Looks good, I've only marked a few minor issues below:

File: pegasus_pt-br.ts (hunk 1 above)
Comment (rutantan) [reply pegasus_pt-br.ts]:
```suggestion
        <translation>Idioma</translation>
```

File: translations.qrc.in (hunk 2 above)
Comment (rutantan) [reply translations.qrc.in]:
```suggestion
        <file>pegasus_pt-br.qm</file>
```

**Comment by rutantan** [comment]
Will create a new pull request as I don't know how to change the file name here.

**Review by rutantan — COMMENTED** [review]
Updates to brazilian portuguese language.

**Review by mmatyas — APPROVED** [review]

**Comment by mmatyas** [comment]
Thanks! Just one last thing, could you merge all the separate commits into a single one?

**Comment by rutantan** [comment]
How can I do that?

Em ter, 4 de dez de 2018 às 08:53, Mátyás Mustoha <notifications@github.com>
escreveu:

**Comment by mmatyas** [comment]
If you're using Git from the terminal, you can `git rebase -i HEAD~5` (`i` for interactive editing, of `HEAD~5`, the last five commits). An editor will pop up, and you should mark all commits other than the first one as fixup/`f`, like this:

screenshot_2018-12-05_14-39-57

Save, then `git push --force`.

If you're using a GUI, there should be similar rebasing tools in it too.

Alternatively I can also do this merging for you if you wish.

**Comment by rutantan** [comment]
Thank you!
I've done it, it should be merged by now.

Em qua, 5 de dez de 2018 às 11:47, Mátyás Mustoha <notifications@github.com>
escreveu:

**Comment by mmatyas** [comment]
Ok, language added, will appear in the next update of Pegasus. Thanks again!

**Pull request merged**